tests/sentry/api/endpoints/test_user_authenticator_enroll.py

```diff
@@ -14,9 +16,18 @@
     Organization,
     OrganizationMember,
 )
+from sentry.utils.compat import mock
 from sentry.testutils import APITestCase
 
 
+# TODO(joshuarli): move all fixtures to a standard path relative to gitroot,
```

Comment:
We should address this TODO as soon as possible as this current approach seems quite fragile. I was wondering if we could [use snapshots](https://github.com/syrusakbary/snapshottest#usage-with-pytest) instead of providing the file manually but I guess we are not using them yet.

Finally, pytest seems to provide the rootdir via `pytest.config.rootdir` but not sure how to use that from tests. Hope this helps.

Reply:
Yes! I was looking for something like rootdir.